src/OpenSpaceToolkit/Astrodynamics/EventCondition/RealCondition.cpp

```diff
@@ -49,14 +53,13 @@ void RealCondition::print(std::ostream& anOutputStream, bool displayDecorator) c
 
     EventCondition::print(anOutputStream, false);
     ostk::core::utils::Print::Line(anOutputStream) << "Criterion: " << StringFromCriterion(getCriterion());
-    ostk::core::utils::Print::Line(anOutputStream) << "Target: " << getTarget();
 
     displayDecorator ? ostk::core::utils::Print::Footer(anOutputStream) : void();
 }
 
 Real RealCondition::evaluate(const State& aState) const
 {
-    return this->evaluator_(aState) - target_;
+    return this->evaluator_(aState) - (target_.value_ + target_.valueOffset_);
```

Comment:
```suggestion
    return this->evaluator_(aState) - (target_.getValue() + target_.getValueOffset());
```

nit: similar comment here to the one in `AngularCondition.cpp`. I think our convention is to have getters/accessors for underscored class variables.